### PR TITLE
Tusk for lesson 7 is done

### PR DESCRIPTION
**Pull request by dero5001**
Plese check my tusk.

### CODE REVIEW

File: Lesson_7/Tusk.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        phone_nr = input('Please enter a number: ')
+        if phone_nr.isdigit():
+            phone_list[name] = phone_nr
+            print(f'The name {name} added')
+        else:
+            print('Only number required, please try again')
+
+    elif 'delete ' in command:
+        del_name = command.split()[1]
+        if del_name[0] == '<' and del_name[-1] == '>':
```
Comment (klymenok) [inline Lesson_7/Tusk.py:36]: I didn't get this part: why do you need these symbols around the name?

**Review by klymenok — APPROVED** [review]
Looks good, but with one comment

File: Lesson_7/Tusk.py (hunk 1 above)
Comment (dero5001) [reply Lesson_7/Tusk.py:36]: The command in tusk was with this simbols. I thought they are important part of the tusk.

Comment (dero5001) [reply Lesson_7/Tusk.py:36]: Also that was interesting to practice how to use some text inside of a string.

**Pull request merged**